README.md

```diff
@@ -27,6 +33,17 @@ To check if everything worked fine, you can do:
 
 and it should not give any error messages.
 
+To run the tests using `pytest`, you need to have the cloned mne-python repository
+with a local pip-install instead of the mne-python package from pypi. Update your
+installation as follows:
+
+	$ git clone https://github.com/mne-tools/mne-python
+	$ cd mne-python
+	$ conda activate mne
```

Comment:
this is confusing unless you also say that `environment.yml` from the installation page of MNE must be used

Reply:
actually, the `environment.yml` in our mne-bids has `mne` as its name :-) so installing the mne-bids env.yml will give you an mne environment. That almost got me into trouble with my mne-python environment. Should I perhaps adjust the mne-bids environment.yml?

Reply:
yeah but you need to first fetch the file and create the environment ... so you have 2 steps missing.

Also the `environment.yml` was really to make appveyor work and not necessarily for users